packages/core/src/ceramic.ts

```diff
@@ -224,7 +224,9 @@ export class Ceramic implements CeramicApi {
     const pinStore = modules.pinStoreFactory.createPinStore()
     const localIndex = new LocalIndexApi(
       params.indexingConfig,
+      this.context,
```

Comment:
could we instead pass some function here that prepares "models of interest" based on the stream id?